docs/source/guides/speed.rst

```diff
@@ -67,19 +52,37 @@ Refer to :doc:`Advanced GPU Optimized Training for more details <../advanced/adv
 
 Prefer DDP over DP
 ^^^^^^^^^^^^^^^^^^
-:class:`~pytorch_lightning.strategies.DataParallelStrategy` performs three GPU transfers for EVERY batch:
+:class:`~pytorch_lightning.strategies.dp.DataParallelStrategy` performs 3 GPU transfers for EVERY batch:
 
-1. Copy model to device.
-2. Copy data to device.
-3. Copy outputs of each device back to main device.
+1. Copy model to the device.
```

Comment:
if you are adding the article there, why not add the other missing one in the sentence? either go all the way or don't bother is what I would say haha ^^

Reply:
Grammarly didn't show it before and not it is 😆